Templates/Java/requests_extensions/BaseEntityCollectionWithReferencesRequest.java.tt

```diff
@@ -45,44 +45,44 @@ import <#=importNamespace#>.concurrency.IExecutors;
     }
 
 <# if (c.GetFeatures().CanExpand) { #>
-    public <#=c.ITypeCollectionRequest()#> expand(final String value) {
+    public <#=c.ITypeCollectionWithReferencesRequest()#> expand(final String value) {
```

Comment:
Looks like a breaking change.

Reply:
on paper it is. I think that:
1. the cast never properly worked, either returning an invalid cast exception or up casting to the first common ancestor (BaseCollectionRequest).
2. the two types share a lot of methods in common, so even if somebody is referencing something, they should get a transparent switch in most cases.

Reply:
Since it is already broken, I agree that we don't need to tag it as a breaking change.